src/devices/Ssd1351/Ssd1351.cs

```diff
@@ -0,0 +1,325 @@
+                    aryTmp[i * 3 + 0] = colourBytes[0];
+                    aryTmp[i * 3 + 1] = colourBytes[1];
+                    aryTmp[i * 3 + 2] = colourBytes[2];
+                }
+            }
+
+            // specifiy a location for the rows and columns on the display where the data is to be written
+            SendCommand(new SetColumnAddress(x, (byte)(x + w - 1)));
+            SendCommand(new SetRowAddress(y, (byte)(y + h - 1)));
+
+            // write out the pixel data
+            SendCommand(Ssd1351Command.WriteRam, aryTmp);
+
+        }
+
+        public void ClearScreen()
+        {
+            FillRect(Color.Black, 0, 0, SCREEN_HEIGHT_PX, SCREEN_WIDTH_PX);
+        }
+
+        /// <summary>
+        /// Resets the display.
+        /// </summary>
+        public async Task ResetDisplayAsync()
+        {
+            _gpioDevice.Write(_dcPinId, PinValue.Low);
+            _gpioDevice.Write(_resetPinId, PinValue.High);
+
+            await Task.Delay(20);
+
+            _gpioDevice.Write(_resetPinId, PinValue.Low);
+            await Task.Delay(20);
+
+            _gpioDevice.Write(_resetPinId, PinValue.High);
+            await Task.Delay(20);
+        }
+
+        /// <summary>
+        /// Verifies value is within a specific range.
+        /// </summary>
+        /// <param name="value">Value to check.</param>
+        /// <param name="start">Starting value of range.</param>
+        /// <param name="end">Ending value of range.</param>
+        /// <returns>Determines if value is within range.</returns>
+        internal static bool InRange(uint value, uint start, uint end)
+        {
+            return (value - start) <= (end - start);
+        }
+
+        /// <summary>
+        /// Send a command to the display controller.
+        /// </summary>
+        /// <param name="command">The command to send to the display controller.</param>
+        public void SendCommand(ICommand command)
+        {
+            if (command == null)
+            {
+                throw new ArgumentNullException(nameof(command));
+            }
+
+            if (command.GetBytes() == null || command.GetBytes().Length == 0)
+            {
+                throw new ArgumentException("The command did not contain any bytes to send.");
+            }
+
+            SendCommand(command.GetBytes());
+
+        }
+
+        /// <summary>
+        /// Send a command to the the display controller.
+        /// </summary>
+        /// <param name="command">Command to send.</param>
+        public void SendCommand(Ssd1351Command command)
+        {
+            SendSPI(new byte[] { (byte)command }, true);
+        }
+
+        /// <summary>
+        /// Send a command to the the display controller.
+        /// </summary>
+        /// <param name="command">Command to send.</param>
+        /// <param name="data">Byte array to send as parameters for the command.</param>
+        public void SendCommand(Ssd1351Command command, byte[] data)
```

Comment:
In order to be able to apply the feedback I gave about the use of Spans, you should change this method to take in Span<byte> instead of byte[].

Reply:
@joperezr 
Done in the latest PR update